<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LandmarksDialog</class>
 <widget class="QDialog" name="LandmarksDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="sName">
       <property name="text">
        <string>Source: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="tName">
       <property name="text">
        <string>Target:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reloadButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="topo-blend.qrc">
         <normaloff>:/images/update.png</normaloff>:/images/update.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Landmarks</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="8" column="0">
        <widget class="QPushButton" name="loadButton">
         <property name="text">
          <string>Load</string>
         </property>
         <property name="icon">
          <iconset resource="topo-blend.qrc">
           <normaloff>:/images/openGraph.png</normaloff>:/images/openGraph.png</iconset>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Save</string>
         </property>
         <property name="icon">
          <iconset resource="topo-blend.qrc">
           <normaloff>:/images/saveGraph.png</normaloff>:/images/saveGraph.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Point landmarks</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QPushButton" name="addPointLandmarkButton">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QListWidget" name="pointLandmarksList">
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="visAllButton">
            <property name="text">
             <string>Vis. All</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="removePointLandmarkButton">
            <property name="text">
             <string>Remove</string>
            </property>
            <property name="icon">
             <iconset resource="topo-blend.qrc">
              <normaloff>:/images/cross-red.png</normaloff>:/images/cross-red.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>File I/O</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QTableWidget" name="landmarksTable">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="columnCount">
          <number>2</number>
         </property>
         <column/>
         <column/>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="tClearButton">
         <property name="text">
          <string>Clear Selections</string>
         </property>
         <property name="icon">
          <iconset resource="topo-blend.qrc">
           <normaloff>:/images/cross-red.png</normaloff>:/images/cross-red.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Part landmarks</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="sClearButton">
         <property name="text">
          <string>Clear Selections</string>
         </property>
         <property name="icon">
          <iconset resource="topo-blend.qrc">
           <normaloff>:/images/cross-red.png</normaloff>:/images/cross-red.png</iconset>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QListWidget" name="sList">
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QListWidget" name="tList">
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>Add</string>
         </property>
         <property name="icon">
          <iconset resource="topo-blend.qrc">
           <normaloff>:/images/downarrow.png</normaloff>:/images/downarrow.png</iconset>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="removeButton">
         <property name="text">
          <string>Remove</string>
         </property>
         <property name="icon">
          <iconset resource="topo-blend.qrc">
           <normaloff>:/images/uparrow.png</normaloff>:/images/uparrow.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Alignment</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="11" column="1" colspan="4">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="3" colspan="2">
        <widget class="QPushButton" name="rotateButton">
         <property name="text">
          <string>Rotate</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Scale</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QSpinBox" name="graphID">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="2" colspan="3">
        <widget class="QSlider" name="scaleTarget">
         <property name="minimum">
          <number>25</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Graph ID</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QSpinBox" name="angleAroundZ">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="minimum">
          <number>90</number>
         </property>
         <property name="maximum">
          <number>270</number>
         </property>
         <property name="singleStep">
          <number>90</number>
         </property>
         <property name="value">
          <number>90</number>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Angle</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="showAABB">
         <property name="text">
          <string>Show AABB</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3" colspan="2">
        <widget class="QPushButton" name="normalizeButton">
         <property name="text">
          <string>Normalize</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="4">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Alignment opertaions</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Source</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Target</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QListWidget" name="alignList1">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QListWidget" name="alignList2">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QPushButton" name="basicAlignButton">
            <property name="text">
             <string>Basic alignment</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Correspondences</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="4" column="2" colspan="2">
        <widget class="QPushButton" name="computeCorrButton">
         <property name="styleSheet">
          <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Cmpt. Corr.</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Score Threshold</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="4">
        <widget class="QTableWidget" name="corrTable">
         <property name="selectionMode">
          <enum>QAbstractItemView::ContiguousSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerItem</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="columnCount">
          <number>3</number>
         </property>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
       <item row="9" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Experiments</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>sourceID</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Visualize Fuzzy Distance</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSpinBox" name="sourceID">
            <property name="maximum">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QPushButton" name="p2pButton">
            <property name="text">
             <string>Test Point2Point Corr.</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QDoubleSpinBox" name="spatialWeight">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="structuralWeight">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Spatial</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Structural</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QDoubleSpinBox" name="orientationWeight">
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2">
        <widget class="QPushButton" name="computeDisMButton">
         <property name="text">
          <string>Cmpt. disM</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="scoreThreshold">
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
         <property name="value">
          <double>0.400000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QDoubleSpinBox" name="sizeWeight">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Size</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Orientation</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2" colspan="2">
        <widget class="QPushButton" name="alignNodesButton">
         <property name="text">
          <string>Align Nodes</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QPushButton" name="runButton">
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="prepareButton">
         <property name="text">
          <string>Prepare Matrices</string>
         </property>
        </widget>
       </item>
       <item row="9" column="3">
        <widget class="QPushButton" name="loadCorrButton">
         <property name="text">
          <string>Load...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="topo-blend.qrc"/>
 </resources>
 <connections/>
</ui>
